orchestra/models.py

```diff
@@ -265,7 +268,7 @@ class Status:
         (Status.PROCESSING, 'Processing'),
         (Status.SUBMITTED, 'Submitted'))
 
-    start_datetime = models.DateTimeField(auto_now_add=True)
+    start_datetime = models.DateTimeField(default=timezone.now)
```

Comment:
oh woops---this comment applies to the new field on Task---what should we do to old Tasks?